gptel-transient.el

```diff
@@ -247,18 +247,23 @@ which see."
       (if (not (file-readable-p gptel-crowdsourced-prompts-file))
           (progn (message "No crowdsourced prompts available")
                  (call-interactively #'gptel-system-prompt))
-        (with-temp-buffer
-          (insert-file-contents gptel-crowdsourced-prompts-file)
-          (goto-char (point-min))
-          (forward-line 1)
-          (while (not (eobp))
-            (when-let ((act (read (current-buffer))))
-              (forward-char)
-              (save-excursion
-                (while (re-search-forward "\"\"" (line-end-position) t)
-                  (replace-match "\\\\\"")))
-              (when-let ((prompt (read (current-buffer))))
-                (puthash act prompt gptel--crowdsourced-prompts)))
+	(with-temp-buffer
+	  (insert-file-contents gptel-crowdsourced-prompts-file)
+	  (goto-char (point-min))
+	  (forward-line 1)
+	  (while (not (eobp))
+            (let ((line (thing-at-point 'line t)))
```

Comment:
Avoid `thing-at-point`, we don't want to load `thingatpt.el` unless it's required.  You can get the line via

```elisp
(buffer-substring-no-properties (line-beginning-position) (line-end-position))
```